docker-compose.yml

```diff
@@ -42,6 +42,9 @@ services:
       - DOCKER_CONFIG_PATH
       - CHT_COMPOSE_PATH
       - HEALTHCHECK_LOG_LEVEL
+      - DB1_DATA
+      - DB2_DATA
+      - DB3_DATA
```

Comment:
Can you please also update the readme to explain what these are?

Reply:
excellent thinking!  added in 446a31d